R/InMemoryAnnData.R

```diff
@@ -118,6 +120,25 @@ InMemoryAnnData <- R6::R6Class("InMemoryAnnData", # nolint
         private$.var_names <- private$.validate_obsvar_names(value, "var")
         self
       }
+    },
+    #' @field obsm
+    obsm = function(value) {
+      if (missing(value)) {
+        private$.obsm
```

Comment:
Add trackstatus comment

```suggestion
        # trackstatus: class=InMemoryAnnData, feature=get_obsm, status=done
        private$.obsm
```